ui/modal/modalPublishPreview/view.jsx

```diff
@@ -260,6 +274,7 @@ const ModalPublishPreview = (props: Props) => {
                   <tbody>
                     {!livestream && !isMarkdownPost && createRow(__('File'), getFilePathName(filePath))}
                     {livestream && remoteFile && createRow(__('Replay'), __('Remote File Selected'))}
+                    {livestream && filePath && createRow(__('Replay'), __('Manual Upload'))}
```

Comment:
:grey_question: Just curious, what are the steps to get this scenario?  Been trying, but never hit it.

Reply:
Edit an existing livestream claim. Manually upload a file instead of choosing a replay from the list.